app/src/main/java/chat/rocket/android/authentication/loginoptions/ui/LoginOptionsFragment.kt

```diff
@@ -120,7 +119,7 @@ fun newInstance(
             putInt(TOTAL_SOCIAL_ACCOUNTS, totalSocialAccountsEnabled)
             putBoolean(IS_LOGIN_FORM_ENABLED, isLoginFormEnabled)
             putBoolean(IS_NEW_ACCOUNT_CREATION_ENABLED, isNewAccountCreationEnabled)
-            putParcelable(DEEP_LINK_INFO, deepLinkInfo)
+            putParcelable(Constants.DEEP_LINK_INFO, deepLinkInfo)
```

Comment:
Does this mean that you are also using persistent storage for the temporary deepLinkInfo..... same as Shailesh did?

Reply:
This code is in the veranda branch. It supports the authentication deep link feature. The only change i made was to consolidate the string constant into Constants, which was defined all over the place. Lastly, this code is the way that you pass arguments into a newly created Fragment - no persistent storage involved.

Reply:
what does putParcelable() do?  It looks to me like you're writing to the Constants file here.